modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/A11yTool.testcase

```diff
@@ -304,6 +304,47 @@ definition {
 		takeScreenshot();
 	}
 
+	@description = "LPS-142333. To assert that filters reset to default after page refresh."
+	@priority = "2"
+	test FiltersAreResetToDefaultAfterPageRefresh {
+		property osgi.module.configuration.file.names = "com.liferay.frontend.js.a11y.web.internal.configuration.FFA11yConfiguration.config";
+		property osgi.module.configurations = "enable=B&quot;true&quot;";
+
+		// Workaround for side panel sometimes not displaying LPS-141442
+
+		while (IsElementNotPresent(locator1 = "A11yTool#VIOLATION_PANEL_HEADER")) {
+			Refresh();
+		}
+
+		task ("Set filter to critical.") {
+			A11yTool.setFilter(filterType = "Critical");
+
+			VerifyElementPresent(
+				key_impact = "critical",
+				locator1 = "A11yTool#VIOLATION_PANEL_ITEM_IMPACT");
```

Comment:
VerifyElementPresent doesn't necessarily check that a checkbox is checked. I'm pretty sure whether the checkbox is checked or not that this step will pass. You'll want to inspect the element and see what changes when you enable/disable the checkbox and then verify that that xpath is present.

Reply:
I don't see any changes to address my concern here, can you check on this? If need help please let me know.

Reply:
@john-co I put the IsChecked function to see if the checkbox is checked. Do you want me to remove the VerifyElementPresent, I kept it to make sure that the text is there.

Reply:
@john-co I kept VerifyElementPresent just to make sure the text was there, but I added the IsChecked function to assert that the checkbox is checked. Do you want me to remove the VerifyElementPresent?